src/__integ__/__snapshots__/themes.test.ts.snap

```diff
@@ -542,7 +542,6 @@ Object {
   "space-panel-split-top": "0px",
   "space-s": "12px",
   "space-scaled-2x-l": "12px",
-  "space-scaled-2x-s": "4px",
```

Comment:
I didn't touch anything in here. I don't know why the snapshots needed to be updated.